### PR TITLE
File drop support

### LINKED ISSUE #4244: [REQUEST]: Allow drag dropping sourcefiles to Tree
### Is your feature request related to a problem? Please describe

I'm sure this is possible, right?

### Describe the solution you'd like

-

### Describe alternatives you've considered

-

### Additional context

Not applicable

### PR DESCRIPTION
**Pull request by partouf**
Initial working version.
Fixes #4244

### CODE REVIEW

**Comment by RubenRBS** [comment]
Will check tomorrow, sorry for the delay!

**Review by RubenRBS — APPROVED** [review]
Oh wow nice! Great job :)

I've changed some _.finds and _.filters to their built-in counterparts, which I think is ok?

File: static/multifile-service.ts [hunk 1]
```diff
@@ -470,12 +487,29 @@ export class MultifileService {
         return suggestedFilename;
     }
 
-    private fileExists(filename: string, excludeFile: MultifileFile): boolean {
-        return !!_.find(this.files, (file: MultifileFile) => {
-            return file !== excludeFile && file.filename === filename;
+    public fileExists(filename: string, excludeFile?: MultifileFile): boolean {
+        return this.files.some((file: MultifileFile) => {
```
Comment (RubenRBS) [inline static/multifile-service.ts:491]: And this too

**Comment by partouf** [comment]
live

**Pull request merged**